### ISSUE TITLE
I used Actnn in Deeplab v3 plus ResNet50, and the it degrade the iou by %2

### ISSUE THREAD

**Issue by hewiew** [ucbrise/actnn#29]

**Comment by hewie33** [comment]
> That's strange. ActNN L0 and full precision training should have identical behavior.
> 
> Could you try to debug by the following:
> 
>     1. prepare a model checkpoint, and a batch of (data, label)
> 
>     2. compute the gradient with full-precision training
> 
>     3. compute the gradient with actnn
> 
>     4. check if the gradients are identical, if not, replace actnn layers with nn.module layers one by one, and observe which layer is causing the gradient discrepancy.
> 
> 
> If you spot a bug in our implementation, please create a PR for us.

Thanks for your advise last week. I've followed the steps you list but it seems we are getting into a trouble:

I restored my model (only a nn.Conv2D layer) from a fixed checkpoint, and I found that the gradient with actnn(set level: "L0") and full-precision training are totally different.

I've sent an email to your Gmail mailbox with my experimental code and printed results with details. You may check that during your free time.
